test/e2e/servicebindingrequest_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/coreos/etcd-operator/pkg/apis/etcd/v1beta2"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
In the other files we've been using `metav1` instead of just `v1`. Please consider.

Reply:
Fixed